### ISSUE TITLE
Support flagging syntax errors as warnings

### ISSUE THREAD

**Issue by sei-areuter** [adrienverge/yamllint#789]
Hi, I know that there is currently documentation that says that since helm isn't valid syntax it will be flagged in yamllint. I was wondering if instead of being able to suppress these errors we could level them out to warnings instead in our `.yamllint`. So an error like this 

```bash
charts/templates/hpa.yaml
  1:3       error    syntax error: expected the node content, but found '-' (syntax)
```

Could be converted to into 

```bash
charts/templates/hpa.yaml
  1:3       warning    syntax error: expected the node content, but found '-' (syntax)
```

With the following .yamllint.yaml 
```bash
rules:
  syntax:
    level: warning
```

I've seen tickets about this before and there's issues with suppressing them before. But I haven't seen anyone request changing the error level before
